src/stores/b2b-purchase-order-flow.md

```diff
@@ -0,0 +1,83 @@
+---
+b2b_only: true
+title: Purchase order workflow
+group: getting-started
+---
+
+Purchase orders allow companies to track and control spending. When purchase orders are enabled for a company account, all orders are automatically processed as purchase orders.
+
+The purchase order workflow can vary in a few ways:
+
+- If no approval rules are set, purchase orders can be placed and the order completed directly.
```

Comment:
The order may not complete directly. It depends what kind of payment is used in the purchase order. If it is offline payment used, it will prompt enter payment details after the purchase order is approved. Here should say: `If no approval rules are set, purchase orders can be placed and the purchase orders will be approved automatically.`

Reply:
It could be true though. Never mind. I think it is fine to list this as a possible flow.